src/ekobox/info_file.py

```diff
@@ -79,5 +79,5 @@ def build(
         dtype=float,
     )
     template_info["AlphaS_Vals"] = alphas_values.tolist()
-    template_info["AlphaS_Qs"] = np.array(operators_card.mu2grid).tolist()
+    template_info["AlphaS_Qs"] = np.sqrt(np.array(operators_card.mu2grid)).tolist()
```

Comment:
```suggestion
    template_info["AlphaS_Qs"] = np.sqrt(operators_card.mu2grid).tolist()
```

Try not to be redundant. `np.sqrt` should act element-wise on arrays and everything that can be turned into an array, yielding always an array as a result.

Reply:
How about using `mugrid` in the first place without any `sqrt`? https://github.com/NNPDF/eko/blob/9321633d106e5814c9d556f16452d7650ab2f8d1/src/eko/io/runcards.py#L223

Reply:
Sorry @felixhekhorn, you're perfectly right!